packages/sirius-web/frontend/sirius-web-application/src/views/project-settings/ProjectSettingsView.tsx

```diff
@@ -28,21 +36,55 @@ const useProjectSettingsViewStyles = makeStyles((theme) => ({
     paddingTop: theme.spacing(3),
     paddingBottom: theme.spacing(3),
   },
+  header: {
+    display: 'grid',
+    gridTemplateColumns: '1fr max-content',
+    alignItems: 'center',
+    padding: theme.spacing(3),
+  },
+  tabs: {
+    display: 'grid',
+    gridTemplateRows: 'min-content 1fr',
+  },
+  tabPanels: {
+    padding: theme.spacing(3),
+  },
 }));
 
 export const ProjectSettingsView = () => {
   const classes = useProjectSettingsViewStyles();
+  const { data: projectSettingsTabContributions } = useData(projectSettingsTabExtensionPoint);
+  const { projectId } = useParams<ProjectSettingsParams>();
+  const [value, setValue] = useState(0);
```

Comment:
This state belongs in the URL

Reply:
In order to merge it a bit more quickly, we will handle this later, I'll just prepare some things for that.